### ISSUE TITLE
why instance can't auto-delete

### ISSUE THREAD

**Issue by slievrly** [alibaba/nacos#759]
![image](https://user-images.githubusercontent.com/8758457/52712456-940b1200-2fcf-11e9-8fb7-4ffb1cc136d4.png)
why instance can't auto-delete, and how to manual delete useless instance。

**Comment by nkorange** [comment]
This is because the ip format is incorrect so Nacos doesn't recognize it thus cannot delete it. We will fix it at #756 .

For now you can use open API to delete this IP:
```
curl  -X DELETE '$nacos_server_ip:8848/nacos/v1/ns/instance?serviceName=serverAddr&ip=localhost&port=8091&clusterName=default'
```

**Comment by slievrly** [comment]
@nkorange thank u , i will try
